actions/bump-version/README.md

```diff
@@ -1,56 +1,51 @@
 # bump-version
-This action will bump the version of your project according to the `.bumpversion.cfg` file.
 
+This action will bump the version of your project according to the `.bumpversion.cfg` file.
 
 ## Prerequisites
-You need a `.bumpversion.cfg` file in the root directory of your repository. A minimal configuration could look like this:
+
+You need a `.bumpversion.cfg` file in `working-directory` (repository root by default). A minimal configuration could look like this:
+
 ```cfg
 [bumpversion]
 current_version = 0.0.1
-commit = True
-tag = True
+commit = False
+tag = False
```

Comment:
Why did you change this?

Reply:
I thought it would be less confusing for the example in the readme, since this action doesn't do any of these things. I also added a disclaimer underneath to link to the `commit-and-push` action. Of course it can still be set to true. Are these two settings actually required? Otherwise I would just remove them altogether.

Reply:
I'm not sure if they are required, but I think removing them is a good idea if you test that it still works then

Reply:
Can confirm, it works without those options set.